app/views/layouts/overdue.html.erb

```diff
@@ -0,0 +1,100 @@
+<% content_for(:content) do %>
+  <!---------------------
+  MOBILE NAV
+  --------------------->
+  <div class="mobile">
+    <div class="mt-0 mb-4 active-filters-mobile">
+      <a class="filters-modal-toggle" data-toggle="modal" data-target="#filter-modal">
+          <span class="filter-tags">
+            <% (Array(@patient_search_filters)).each do |filter, label| %>
+              <span class="badge badge-pill badge-secondary"><%= label %></span>
+            <% end %>
+          </span>
+        <i class="fas fa-filter"></i> Search filters
+      </a>
+    </div>
+    <!-- Filter Modal -->
+    <div class="modal fade" id="filter-modal" tabindex="-1" role="dialog" aria-labelledby="filter-modal" aria-hidden="true">
+      <div class="modal-dialog modal-dialog-centered" role="document">
+
+        <div class="modal-content">
+          <div class="modal-header">
+            <h2 class="modal-title" id="exampleModalCenterTitle">Patients</h2>
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body">
+            <div class="row">
+              <div class="col-6 mb-3">
+                <h4>Patients</h4>
+                <%= form_tag appointments_path, method: :get do %>
+                  <% @search_filters.each do |filter, label| %>
+                    <div class="checkbox-row">
+                      <label for=<%= "search-#{filter}-mobile" %>>
+                        <% checked = @search_filters.include?(filter) %>
+                        <%= check_box_tag("search_filters[]", filter, checked, { id: "search-#{filter}-mobile" } ) %>
+                        <%= label %>
+                      </label>
+                    </div>
+                  <% end %>
+                <% end %>
+              </div>
+            </div>
+          </div>
+          <div class="modal-footer">
+            <a href="?" type="button" class="btn btn-secondary">Clear all</a>
```

Comment:
Is there a Clear All link in the desktop version? Does this link trigger a page refresh with empty query parameters? Will that end up populating the default search params? We could use `?submitted=true` if we want to avoid that.

Reply:
There is not, this looks like a mobile only thing that I took from my facilities.  I'll test this out to make sure it resets to the defaults.